README.md

```diff
@@ -54,46 +52,38 @@ Requirements:
 * Each standard can have a separate directory no deeper than 3 levels from the repository root.
 * The package `type` must be `phpcodesniffer-standard`. Without this, the plugin will not trigger.
 
-[this]: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Coding-Standard-Tutorial
-
 ## Contributing
 
-This is an active open-source project. We are always open to people who want to use the code or contribute to it.
+This is an active open-source project. We are always open to people who want to
+use the code or contribute to it.
 
-We've set up a separate document for our [contribution guidelines].
+We've set up a separate document for our [contribution guidelines][contributing-guidelines].
 
 Thank you for being involved! :heart_eyes:
 
-[contribution guidelines]: https://github.com/dealerdirect/phpcodesniffer-composer-installer/blob/master/CONTRIBUTING.md
-
-## Authors & Contributors
+## Authors & contributors
 
-The original idea and setup of this repository is by [Franck Nijhof], employee @ Dealerdirect.
+The original idea and setup of this repository is by [Franck Nijhof][frenck], employee @ Dealerdirect.
 
-For a full list off all author and/or contributors, please check [this page].
+For a full list off all author and/or contributors, please check [this page][contributors].
```

Comment:
Please avoid using "this" for links. Suggestion:

    For a full list off all author and/or contributors, please check [the contributors page][contributors].